gensim/models/doc2vec.py

```diff
@@ -609,8 +609,9 @@ def __init__(self, documents=None, corpus_file=None, dm_mean=None, dm=1, dbow_wo
             self.build_vocab(documents=documents, corpus_file=corpus_file,
                              trim_rule=trim_rule)
             self.train(
-                documents=documents, corpus_file=corpus_file, total_examples=self.corpus_count, epochs=self.epochs,
-                start_alpha=self.alpha, end_alpha=self.min_alpha, callbacks=callbacks)
+                documents=documents, corpus_file=corpus_file, total_examples=self.corpus_count,
+                total_words=self.corpus_total_words, epochs=self.epochs, start_alpha=self.alpha,
```

Comment:
`train()` needs either `total_words` OR `total_examples`, but not both – currently only documented in the deep-private method https://github.com/RaRe-Technologies/gensim/blob/27c524db60828e5ab1580a46a8ca5520dfd4d352/gensim/models/base_any2vec.py#L1041 . Further if `total_examples` *is* supplied, `total_words` won't even be consulted (though this is quite obscure in the current code layout, only evident in the deep-abstract-private method https://github.com/RaRe-Technologies/gensim/blob/27c524db60828e5ab1580a46a8ca5520dfd4d352/gensim/models/base_any2vec.py#L250 ).

Reply:
I don't think this note relates to current PR, because I didn't change this code. It exists in the current develop branch. Do you want me to fix it somehow in current PR?

Reply:
Sorry, I only noticed it as a commit this PR, and its change (adding `total_words`) was wholly unnecessary. You may ignore.